.github/workflows/testBuild.yml

```diff
@@ -140,39 +159,7 @@ jobs:
           APPLE_ID_PASSWORD: ${{ secrets.APPLE_ID_PASSWORD }}
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-
-  # web:
-  #   name: Build and deploy Web
-  #   needs: validateActor
-  #   if: ${{ fromJSON(needs.validateActor.outputs.IS_TEAM_MEMBER) }}
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #     - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
-  #       with:
-  #         fetch-depth: 0
-  #         ref: ${{ github.event.pull_request.head.sha }}
-
-  #     - uses: Expensify/App/.github/actions/composite/setupNode@main
-
-  #     - name: Configure AWS Credentials
-  #       # Version: 1.5.5
-  #       uses: aws-actions/configure-aws-credentials@e97d7fbc8e0e5af69631c13daa0f4b5a8d88165b
-  #       with:
-  #         aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-  #         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-  #         aws-region: us-east-1
-
-  #     - name: Build web for staging
-  #       run: npm run build-staging
-
-  #     - name: Build docs
-  #       run: npm run storybook-build
-  #       continue-on-error: true
-
-  #     - name: Deploy to S3 for internal testing
-  #       run: aws s3 cp --recursive --acl public-read "$GITHUB_WORKSPACE"/dist s3://ad-hoc-expensify-cash/web/"$PULL_REQUEST_NUMBER"
-  #       env:
-  #         PULL_REQUEST_NUMBER: ${{ github.event.number }}
+          PULL_REQUEST_NUMBER: ${{ github.event.number }}
```

Comment:
@staszekscp Is this intentionally here?

Reply:
Yes, I add this environmental variable in order to place the desktop app in a folder associated with the correct PR

Reply:
Oh ok, thanks!